src/main/java/com/yahoo/bullet/bql/extractor/QueryExtractor.java

```diff
@@ -61,6 +65,7 @@
 
     private Set<Expression> groupByFields;
     private Set<Expression> selectFields;
+    private Set<Expression> computations;
```

Comment:
Using Set will mess the sequence of computations, that's not we want. Use List instead.